CHANGELOG.md

```diff
@@ -126,6 +126,7 @@
         * Rename `SEPADirectDebitPaymentAuthRequestCallback#onResult` to
           `SEPADirectDebitPaymentAuthRequestCallback#onSEPADirectDebitPaymentAuthResult`
     * Visa Checkout
+        * Publishing disabled for v5
```

Comment:
Can we just remove this whole CHANGELOG and say `Visa checkout is not yet available for v5` and also remove mirgration guide portion and say the same?

Reply:
yep, that sounds good! let me push an update.